File: app/src/main/java/br/com/dbccompany/sicred/domain/model/CheckIn.java
package br.com.dbccompany.sicred.domain.model;

public class CheckIn {
    String eventId;
    String name;
    String email;


    public String getEventId() {
        return eventId;
    }

    public void setEventId(String eventId) {
        this.eventId = eventId;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }


    public CheckIn(String eventId, String name, String email) {
        this.eventId = eventId;
        this.name = name;
        this.email = email;
    }

    public CheckIn() {
    }
}

File: app/src/main/java/br/com/diogomoreno/events/domain/model/CheckIn.kt
package br.com.diogomoreno.events.domain.model

class CheckIn {
    var eventId: String? = null
    var name: String? = null
    var email: String? = null

    constructor(eventId: String?, name: String?, email: String?) {
        this.eventId = eventId
        this.name = name
        this.email = email
    }

    constructor() {}
}